tests/4.5/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_dist_schedule.c

```diff
@@ -0,0 +1,63 @@
+//===-test_target_teams_distribute_parallel_for_dist_schedule.c-===//
+//
+// OpenMP API Version 4.5 Nov 2015
+//
+// testing the combined construct target teams distribute parallel for
+// This test will check if dist_schedule(static, 4) taking effect
+//
+//===---------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define THRD 10
+#define NUM_TEAMS 2
+
+int Dist_ScheduleStatic() {
+  int ErrCount = 0;
+
+  int TrackUsedThread[NUM_TEAMS][THRD];
+  for (int j = 0; j < NUM_TEAMS; ++j) {
+    for (int i = 0; i < THRD; ++i) {
+      TrackUsedThread[j][i] = 0;
+    }
+  }
+
+#pragma omp target teams distribute parallel for num_teams(NUM_TEAMS)\
+        thread_limit(THRD)  map(tofrom: TrackUsedThread[0:NUM_TEAMS])\
+        dist_schedule(static, 4)
+  for (int i = 0; i < 32; ++i) {
+    TrackUsedThread[omp_get_team_num()][omp_get_thread_num()] += 1;
+  }
+
```

Comment:
Since the number of teams and threads can be different than those requested; use captured values for checking after the target region.

Reply:
Hi @spophale ,
a) I have committed the changes suggested by you
b) I removed using "TrackUsedThread[][]" array which I used earlier for verification
c) My new verification should be the following ? please suggest...
if ((num_teams == 2) && (num_threads == 10)) --> pass
                      or 
if ((num_teams >= 1) && (num_threads >= 1)) --> pass

Reply:
Hi @spophale ,
Please consider the latest changes uploaded....
Thank you!!!